ansible/roles/touchforms/templates/localsettings.py.j2

```diff
@@ -3,7 +3,8 @@
 #
 import os
 
-URL_ROOT = "https://{{ SITE_HOST }}/a/{{ '{{' }}DOMAIN{{ '}}' }}"
+URL_HOST = "https://{{ SITE_HOST }}"
+URL_ROOT = "%s/a/{{ '{{' }}DOMAIN{{ '}}' }}" % URL_HOST
```

Comment:
ah wow thanks for digging into that. makes sense. for now i'll update with `raw` just so i don't mess up other people's localsettings but agree probably a different syntax is best